packages/angular-material/src/other/table.renderer.ts

```diff
@@ -48,12 +48,33 @@ import {
 @Component({
   selector: 'TableRenderer',
   template: `
+
     <table
       mat-table
       [dataSource]="data"
       class="mat-elevation-z8"
       [trackBy]="trackElement"
     >
+      <ng-container matColumnDef="action">
+        <tr>
+          <th mat-header-cell *matHeaderCellDef>
+            <button mat-icon-button color="primary" (click)="add()" [disabled]="!isEnabled()" title="Add"><mat-icon>add</mat-icon></button>
+          </th>
+        </tr>
+        <tr>
+          <td mat-cell *matCellDef="let row; let i = index" style="width:20%">
+            <button
+                mat-button
+                color="warn"
+                (click)="remove(i)"
+                [disabled]="!isEnabled()"
+                matTooltipPosition="right"
```

Comment:
Add tooltip to delete buttons
```suggestion
                matTooltipPosition="right"
                matTooltip="Delete"
```